lib/theme.ts

```diff
@@ -0,0 +1,73 @@
+export function theme(focusBgColor: string, backgroundColor: string) {
+  return {
+    root: {
+      primaryColor: "rgb(8 117 225)",
+      dangerColor: "salmon",
+      warningColor: "yellow",
+    },
+    sidebar: {
+      // TODO: White Logo needs to be added on the config side, then the logo can be removed from here
+      logo: "https://images.ctfassets.net/hjneo4qi4goj/gL6Blz3kTPdZXWknuIDVx/7bb7c73d93b111ed542d2ed426b42fd5/flatfile.svg",
```

Comment:
Can this be put in /public and referenced off of hcm.show/blah?

Reply:
I dont think so. Tried adding the image to public and importing to set it and it doesnt take. Looked at the docs and they only have a URL set for it

Reply:
Going to try from the actual config side. Add in public and set as logo. If thats ok, hopefully adding the theme from the app side doesnt override the whole theme

Reply:
To be clear I'm meaning add to public and then in the config put `https://hcm.show/the-asset.png`. Still an external reference, just not stored on a random image site. Could also put in our google drive folder too

Reply:
Ahh I see. Makes sense to do it this way